crates/jose/src/claims.rs

```diff
@@ -33,16 +32,19 @@ pub enum ClaimError {
     ValidationError {
         claim: &'static str,
         #[source]
-        source: anyhow::Error,
+        source: Box<dyn std::error::Error + Send + Sync + 'static>,
```

Comment:
I originally made that a generic type param, but that was too much of a headache to use